flow/client_server/static/internal/groups/create/js/script.js

```diff
@@ -0,0 +1,36 @@
+function onceDocumentReady() {
+	getURLs(startSubmitButtonListener);
+}
+
+function startSubmitButtonListener() {
+	$('#submit').click(function () {
+		var group_name = $('#group_name').val();
+		var aid = getAID();
+		var data = {
+			group_name: group_name,
+			aid: aid
```

Comment:
It's a key value pair in an object/dictionary.

The key is 'aid', and the value is the value of the variable `aid`.

https://www.w3schools.com/js/js_objects.asp